Application/EdFi.Ods.Api/Security/AuthorizationStrategies/Relationships/Filters/RelationshipsAuthorizationFilters.cs

```diff
@@ -78,14 +72,15 @@ private static FilterApplicationDetails CreateClaimValuePropertyFilter(string pr
                     new FilterApplicationDetails(
                         "SchoolIdToStudentUSI",
                         @"StudentUSI IN (
-                        SELECT {newAlias1}.StudentUSI 
-                        FROM auth.SchoolIdToStudentUSI {newAlias1} 
-                        WHERE {newAlias1}.SchoolId IN (:SchoolId))",
+                            SELECT {newAlias1}.StudentUSI 
+                            FROM auth.StudentUSIToEducationOrganizationId {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:SourceEducationOrganizationId))",
                         @"{currentAlias}.StudentUSI IN (
-                        SELECT {newAlias1}.StudentUSI 
-                        FROM " + "auth_SchoolIdToStudentUSI".GetFullNameForView() + @" {newAlias1} 
-                        WHERE {newAlias1}.SchoolId IN (:SchoolId))",
-                        (c, w, p, jt) => c.ApplyJoinFilter(w, p, "SchoolIdToStudentUSI", "StudentUSI", "SchoolId", jt),
+                            SELECT {newAlias1}.StudentUSI 
+                            FROM " + "auth_StudentUSIToEducationOrganizationId".GetFullNameForView() + @" {newAlias1} 
+                            WHERE {newAlias1}.SourceEducationOrganizationId IN (:LocalEducationAgencyId))",
```

Comment:
Why LocalEducationAgencyId here?  Shouldn't this be SchoolId?

Reply:
Found it.  The Hql conditions are used by Composites only, so this doesn't break unless you test with Composites.  This needs to be fixed to SchoolId, or it will fail in the following scenario: client with access to a School, hitting the sections composite, throws an error because the :LocalEducationAgencyId doesn't get set.

Reply:
I updated it now . Thanks